packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -11,10 +15,19 @@ const {
 const qs = require(`qs`)
 const base64Img = require(`base64-img`)
 const _ = require(`lodash`)
-const path = require(`path`)
 
 const cacheImage = require(`./cache-image`)
 
+// By default store the images in `.cache` but allow the user to override
+// and store the image cache away from the gatsby cache. After all, the gatsby
+// cache is more likely to go stale than the images (which never go stale)
+const REMOTE_CACHE_FOLDER =
```

Comment:
This sort of capability _should_ be built into the core caching APIs. Adding it here could be fine but should be made clear in the comment that this is a one off API & may be removed at any moment.